prinz/src/main/scala/pl/touk/nussknacker/prinz/util/repository/HttpRepositoryClient.scala

```diff
@@ -0,0 +1,43 @@
+package pl.touk.nussknacker.prinz.util.repository
+
+import com.typesafe.scalalogging.LazyLogging
+
+import java.net.URI
+import org.jsoup.Jsoup
+
+import java.io.{File, InputStream}
+import scala.collection.JavaConverters
+
+
+class HttpRepositoryClient(fileExtension: String) extends AbstractRepositoryClient(fileExtension) {
+
+  override protected def readPath(path: URI): Iterable[ModelPayload] = {
+    if(isCorrectFile(path)) {
+      List(ModelPayload(path))
+    }
+    else {
+      val pathString = path.toString
+      try {
```

Comment:
When creating first process, here some error is thrown, but even then all models are listed. And while creating next processes, no errors are thrown. But I couldn't find the problem.